src/main/java/com/gregtechceu/gtceu/api/pattern/MultiblockState.java

```diff
@@ -97,17 +161,26 @@ public IMultiController getController() {
         return null;
     }
 
+    /** See {@link MultiblockState#error} */
     public boolean hasError() {
         return error != null;
     }
 
+    /** See {@link MultiblockState#error} */
     public void setError(PatternError error) {
         this.error = error;
         if (error != null) {
             error.setWorldState(this);
         }
     }
 
+    /**
+     * Returns the {@link BlockState} at the active position.
+     * If null, tries to query it by checking at the in-world pos.
+     * If still null, returns error.
+     * 
+     * @return blockState
+     */
```

Comment:
The first null check is a lazy-cache type deal. The second null check is an actual null check. An error isn't returned. 
It's ok to be concise. You can use the `{@return}` element to duplicate it in the summary and the return field. See how this looks in your IDE
```suggestion
    /**
     * {@return the BlockState at the active position} 
     * {@code null} if the BlockState was unable to be gotten for some reason.
     */
```